lib/orbit_web/controllers/admin/rfid.html.heex

```diff
@@ -0,0 +1,14 @@
+<h1>admin/rfid</h1>
+<p>Manually Create RFID Badge Serial</p>
+
+<.form for={%{}} action={~p"/admin/rfid"} method="post">
+  <label class="block">
```

Comment:
These forms are in a bit of an awkward CSS zone, in that we don't have designs for them and shouldn't care too much, but also tailwind is loaded, which means that default browser styles are disabled. So I ended up doing just a little bit of awkward styles for this and the button, and there's missing styles for stuff that's normally styled like required inputs.

Any ideas on what a better way to handle this would be? Should we fully style this? (On Glides, the admin pages benefit from default styles that orbit doesn't have.) Is there a way to remove the stylesheet from this page so we can use the default browser styles?

Reply:
My initial reaction is that the screenshots don't look perfect but they're fine as-is, though I'm sure there's _some_ way to remove the Tailwind CSS if we really cared to do so.